lib/rubocop/cop/style/empty_line_between_defs.rb

```diff
@@ -21,10 +21,13 @@ def on_def(node)
           add_offense(node, :keyword)
         end
 
+        alias_method :on_defs, :on_def
```

Comment:
We have a mixin called `OnMethodDef` or something like this, which you should be using when having to deal with `def` and `defs` nodes.